src/cow-react/modules/limitOrders/pure/Orders/OrderRow.tsx

```diff
@@ -33,6 +36,11 @@ const RateValue = styled.span`
   font-size: 12px;
 `
 
+const StatusBox = styled.div`
```

Comment:
Just out of curiosity, you think keeping styles in components makes more sense in this case instead of creating one styled file?

Reply:
https://cowservices.slack.com/archives/C0361CDG8GP/p1669113303229269
Discussed it here.
For now, `OrderRow` component is pretty small, so, it makes sense to keep styled elements in the component file.
Join the discussion :) Very interesting to see your opinion